### PR TITLE
Reduce useless resolutions

### PR DESCRIPTION
**Pull request by Et7f3**
Some resolutions might be useless + I didn't update the lockdir (because I am on the github's web editor)
Apply on top of #1082

### CODE REVIEW

**Comment by Et7f3** [comment]
Error in the CI https://dev.azure.com/onivim/oni2/_build/results?buildId=8220&view=logs&j=ca1de7b4-19db-5646-f7fe-ed7c2dd3de8a&t=4542f74c-da99-569e-bca0-ee14ffe6ea2f&l=14 but should be in other issue/PR

File: package.json [hunk 1]
```diff
@@ -216,19 +216,14 @@
...
-    "@opam/js_of_ocaml-ppx": "github:ocsigen/js_of_ocaml:js_of_ocaml-ppx.opam#db257ce",
+    "@opam/js_of_ocaml": "3.5.2",
+    "@opam/js_of_ocaml-compiler": "3.5.0",
+    "@opam/js_of_ocaml-lwt": "3.5.2",
     "rebez": "github:jchavarri/rebez#46cbc183",
     "reasonFuzz": "github:CrossR/reasonFuzz#d36e084",
     "pesy": "0.4.1",
-    "@opam/ocaml-migrate-parsetree": "1.3.1",
-    "@opam/ppx_tools_versioned": "5.2.2",
-    "reason-fontkit": "2.7.0",
```
Comment (bryphe) [inline package.json]: Nice! Thanks @Et7f3 for cleaning this up 🎉

**Comment by CrossR** [comment]
CI failure looks unrelated to this, so could we get this merged once the CLA stuff is sorted with @Et7f3?

I'm hitting issues adding resolutions to various things (i.e. testing the textmate changes) unless I have these changes applied locally.

**Comment by bryphe** [comment]
> CI failure looks unrelated to this, so could we get this merged once the CLA stuff is sorted with @Et7f3?

For this change - I'm OK bringing it w/o a CLA (since it's just a couple of `package.json` changes).

> I'm hitting issues adding resolutions to various things (i.e. testing the textmate changes) unless I have these changes applied locally.

Sure thing, I just checked the latest build and you're right - seems unrelated. I'll override and bring in now 👍

**Pull request merged**